src/CoursePageResources/index.scss

```diff
@@ -0,0 +1,48 @@
+.container-fluid {
+    background-color: #EAF3FB;
+    color: #2E8ED3;
+}
+
+.container-fluid-container {
+    border-color: #B2D5EE;
+}
+
+.container-fluid-enabled-button {
+    border-color: #0075B4;
+    color: #2E8ED3;
+}
+
+.container-fluid-enabled-button:hover {
+    color: inherit;
+}
+
+.sub-container-text {
+    color: #39AAE3;
+}
+
+.border-enabled {
+    border-color: #4293C4 !important; 
+    border-width: 2px;
+    border-style: solid;
+    border-radius: 5px;
+}
+
+.border-disabled {
+    border-color: #ffffff !important;
+    border-width: 2px;
+    border-style: solid;
+    border-radius: 5px;
+}
```

Comment:
Both of these share most of their code. It would be better to have it as:

```scss
.course-page-tile {
  border: 2px solid $white;
  border-radius: 5px; //Maybe this can be standardised?
      .disabled {
          border-coloor: #4293C4;
      }
  }
}